packager/mpd/base/mpd_builder.cc

```diff
@@ -53,12 +53,12 @@ bool AddMpdNameSpaceInfo(XmlNode* mpd) {
       "urn:marlin:mas:1-0:services:schemas:mpd";
   static const char kXmlNamespaceXlink[] = "http://www.w3.org/1999/xlink";
   static const char kMsprNamespace[] = "urn:microsoft:playready";
+  static const char kScte214Namespace[] = "scte214";
```

Comment:
wonder if this is the correct namespace to use @cosmin  ? 
or if this should be `urn:scte:dash:scte214-extensions`
```
4.2.4 Guidelines for signaling cross-compatible Dolby Vision content (profiles
8.1 and 8.4)
Follow the guidelines for MPD schema and AdaptationSet when signaling cross-compatible Dolby Vision
content.
MPD
An MPD that needs to use this signaling must contain the following schema.
xmlns:scte214="urn:scte:dash:scte214-extensions"
```